ci/stage-test-sgx.jenkinsfile

```diff
@@ -105,11 +104,11 @@ stage('test-sgx') {
                 fi
 
                 cd CI-Examples/redis
-                if [ "${gcc_dump_machine}" == 'x86_64-redhat-linux' ]
+                if [ "${os_release_id}" != 'ubuntu' ]
                 then
                     sed -i '$ a loader.pal_internal_mem_size = "64M"' redis-server.manifest.template 
                 fi                
-                make ${MAKEOPTS} ${ARCH_LIB_OPT}
```

Comment:
why did we remove ARCH_LIB_OPT? what was it used for before?

Reply:
It was used for CentOS, Now we don't need it because ARCH_LIBDIR has been declared as environment variable for CentOS inside config-centos8.jenkinsfile